internal/fs/wrappers/error_mapping.go

```diff
@@ -59,16 +60,28 @@ func errno(err error) error {
 	}
 
 	// Cannot authenticate or Permission denied.
-	// The control client API returns an RPC error code instead of googleapi code.
-	if strings.Contains(err.Error(), "oauth2: cannot fetch token") || strings.Contains(err.Error(), "PermissionDenied") {
+	if strings.Contains(err.Error(), "oauth2: cannot fetch token") {
 		return syscall.EACCES
 	}
 
-	// Translate API errors into an em errno
-	var apiErr *googleapi.Error
+	// The control client API returns an RPC error code instead of googleapi code.
+	var apiErr *apierror.APIError
 	if errors.As(err, &apiErr) {
-		switch apiErr.Code {
-		case http.StatusForbidden:
+		switch apiErr.GRPCStatus().Code() {
```

Comment:
can you add a comment why only grpc statuc code is checked here. There is also googleapi error in the apierr definition.

Reply:
Done